server/src/routes/serializer/ParkSerializer.js

```diff
@@ -0,0 +1,27 @@
+class ParkSerializer {
+    static async showData(park) {
+        const allowedAttributes = ["id", "name", "location", "description", "picture"];
+
+        let serializedPark = {};
+        for (const attribute of allowedAttributes) {
+            serializedPark[attribute] = park[attribute];
+        }
+        serializedPark.reviews = await park.$relatedQuery("reviews")
+        for (let i = 0; i < serializedPark.reviews.length; i++) {
+            serializedPark.reviews[i].user = await serializedPark.reviews[i].$relatedQuery("user")
```

Comment:
We will need a ReviewSerializer! And probably a UserSerializer if you're also trying to find the User who made the review